### ISSUE TITLE
Does the loss requires another term?

### ISSUE THREAD

**Issue by DreadPiratePsyopus** [marvis/pytorch-yolo2#113]
Hello @marvis,

Good repo and congrats for it. I'd been through the code and I only have one slight doubt regarding the loss function.
The loss function in region_loss  is:

        loss_x = self.coord_scale * nn.MSELoss(size_average=False)(x*coord_mask, tx*coord_mask)/2.0
        loss_y = self.coord_scale * nn.MSELoss(size_average=False)(y*coord_mask, ty*coord_mask)/2.0
        loss_w = self.coord_scale * nn.MSELoss(size_average=False)(w*coord_mask, tw*coord_mask)/2.0
        loss_h = self.coord_scale * nn.MSELoss(size_average=False)(h*coord_mask, th*coord_mask)/2.0
        loss_conf = nn.MSELoss(size_average=False)(conf*conf_mask, tconf*conf_mask)/2.0
        loss_cls = self.class_scale * nn.CrossEntropyLoss(size_average=False)(cls, tcls)
loss = loss_x + loss_y + loss_w + loss_h + loss_conf + loss_cls

However, according to the yolo paper, don't you think it requires an additional term to address whenever the is no object? It can be something like:

loss_no_obj = nn.MSELoss(size_average=False)(conf*(1-conf_mask), tconf*(1-conf_mask))/2.0

In order to  make confidence score close to 0 when there are no object in the cell.
Let me know your thoughts, I will let you know any training results with this change.
